app/assets/javascripts/components/students/shared/StudentList/StudentDrawer/TrainingStatus/TrainingModuleRows.jsx

```diff
@@ -11,9 +11,17 @@ export const TrainingModuleRows = ({ trainingModules }) => {
   return trainings.map((trainingModule) => {
     let moduleStatus;
     if (trainingModule.completion_date) {
+      let completionTime;
+      if (trainingModule.completion_time <= 60 * 60) {
+        completionTime = moment.utc(trainingModule.completion_time * 1000).format('HH:mm:ss');
+      } else {
+        completionTime = I18n.t('training_status.sufficient_time_allocation');
```

Comment:
I think it should not show this message. Taking a long time between starting and finishing doesn't necessarily mean any significant time was spent on each slide; it could be that they just started it, left it, and then rushed through it later. If the total completion time is very short, that's a meaningful bit of data that we want to show, but if it's long, it's not a clear enough signal to draw any conclusions from.

Reply:
@ragesoss Okay! I just removed the message if it is more than one hour.